test/e2e/v3/plugin_cluster_test.go

```diff
@@ -118,6 +118,24 @@ var _ = Describe("kubebuilder", func() {
 				kbc.UninstallCertManager(false)
 			})
 
+			It("should generate a runnable project go/v3 using kustomize 3.x", func() {
```

Comment:
Should we not be testing for 4.x as we are switching the version from `v3.8.7` to `v4.5.5`?

Reply:
All the other tests checks 4.5.5. The goal of this test is to ensure the backwards compatibility ( not allow contributors to change the scaffold for syntax that are valid only from 4.x )

Reply:
Got it. Thanks for clarifying.